diff --git a/node_modules/@comapeo/nodejs-mobile-react-native/android/build.gradle b/node_modules/@comapeo/nodejs-mobile-react-native/android/build.gradle
index 0798fc8..7d3e9da 100644
--- a/node_modules/@comapeo/nodejs-mobile-react-native/android/build.gradle
+++ b/node_modules/@comapeo/nodejs-mobile-react-native/android/build.gradle
@@ -385,16 +385,27 @@ if ("1".equals(shouldRebuildNativeModules)) {
                     def moduleName = moduleRoot.getName()
                     println "Preparing to use the prebuild in ${moduleName}"
                     delete "${moduleRoot}/build"
-                    copy {
-                        from "${dotNode.getAbsoluteFile()}"
-                        into "${moduleRoot}/build/Release/"
+
+                    // Modules using node-gyp-build expect the .node file to be in the build/Release folder,
+                    // but modules using bare-addon expect it in the prebuilds folder.
+                    def bindingGypFile = file("${moduleRoot}/binding.gyp")
+                    if (bindingGypFile.exists()) {
+                        copy {
+                            from "${dotNode.getAbsoluteFile()}"
+                            into "${moduleRoot}/build/Release/"
+                        }
+                        // Delete the prebuilds since it's not needed at runtime for node-gyp-build modules
+                        delete "${moduleRoot}/prebuilds"
+                        delete "${moduleRoot}/binding.gyp"
+                        println "Moved prebuild .node to ${moduleName}/build/Release/ and deleted binding.gyp"
+                    } else {
+                        println "Skipping move for ${moduleName} - no binding.gyp found"
                     }
+                    // Rename package.json scripts that would trigger a build if npm install is run again.
                     exec {
                         workingDir moduleRoot
                         commandLine 'sed', '-i.bak', 's/"install"/"dontinstall"/g; s/"rebuild"/"dontrebuild"/g; s/"gypfile"/"dontgypfile"/g', 'package.json'
                     }
-                    delete "${moduleRoot}/binding.gyp"
-                    delete "${moduleRoot}/prebuilds"
                 }
             }
         }
