diff --git a/node_modules/@comapeo/nodejs-mobile-react-native/android/build.gradle b/node_modules/@comapeo/nodejs-mobile-react-native/android/build.gradle
index 0686af1..2f17837 100644
--- a/node_modules/@comapeo/nodejs-mobile-react-native/android/build.gradle
+++ b/node_modules/@comapeo/nodejs-mobile-react-native/android/build.gradle
@@ -1,11 +1,12 @@
 
 buildscript {
     repositories {
+        google()
         mavenCentral()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.0'
+        classpath 'com.android.tools.build:gradle:8.13.0'
     }
 }
 
@@ -19,40 +20,17 @@ if (_compileNativeModulesSdkVersion<24) {
     // 24 is the minimum sdk version Node is built with.
     _compileNativeModulesSdkVersion=24;
 }
-def DoesAppAlreadyDefineWantedSTL() {
-    // Since react-native 0.59.0, the Application already defines libc++_shared as the APP_STL.
-    // Defining it also in this plugin would lead to a build error when merging assets.
-    try {
-        def _reactAndroidPropertiesFile = file("${rootDir}/../node_modules/react-native/ReactAndroid/gradle.properties");
-        def _reactAndroidProperties = new Properties()
-        if (_reactAndroidPropertiesFile.exists())
-        {
-            _reactAndroidPropertiesFile.withInputStream { _reactAndroidProperties.load(it) }
-        }
-        def _semver = _reactAndroidProperties.getProperty("VERSION_NAME").tokenize('.');
-        if (_semver.size() != 3) {
-            return false
-        }
-        def _major = _semver[0].toInteger()
-        def _minor = _semver[1].toInteger()
-        if ( _major > 0 || (_major == 0 && _minor >= 59) ) {
-            return true
-        } else {
-            return false
-        }
-
-    } catch ( Exception e ) {
-        return false
-    }
-}
-
-def _isCorrectSTLDefinedByApp = DoesAppAlreadyDefineWantedSTL()
 
 android {
     compileSdkVersion ((rootProject?.ext?.properties?.compileSdkVersion) ?: 24)
-    buildToolsVersion ((rootProject?.ext?.properties?.buildToolsVersion) ?: "30.0.2")
 
-    ndkVersion ((rootProject?.ext?.ndkVersion) ?: "24.0.8215888")
+    if (rootProject.hasProperty("ndkPath")) {
+        ndkPath rootProject.ext.ndkPath
+    }
+
+    if (rootProject.hasProperty("ndkVersion")) {
+        ndkVersion rootProject.ext.ndkVersion
+    }
 
     defaultConfig {
         minSdkVersion _nodeMinSdkVersion
@@ -62,12 +40,10 @@ android {
         externalNativeBuild {
             cmake {
                 cppFlags ""
-                // Enable 16 KB ELF alignment (NDK r27 and higher)
-                // https://developer.android.com/guide/practices/page-sizes#compile-r27
-                arguments "-DANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES=ON"
-                if(!_isCorrectSTLDefinedByApp) {
-                    arguments "-DANDROID_STL=c++_shared"
-                }
+                arguments "-DANDROID_STL=c++_shared", 
+                        // Enable 16 KB ELF alignment (NDK r27 and higher)
+                        // https://developer.android.com/guide/practices/page-sizes#compile-r27
+                        "-DANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES=ON"
             }
         }
         ndk {
@@ -308,7 +284,7 @@ if ("1".equals(shouldRebuildNativeModules)) {
             throw new GradleException("Unsupported operating system for nodejs-mobile native builds: ${OperatingSystem.current().getName()}")
         }
 
-        String ndk_bundle_path = android.ndkDirectory
+        String ndk_bundle_path = android.ndkPath
         String toolchain_path = "${ndk_bundle_path}/toolchains/llvm/prebuilt/${temp_host_tag}"
         String npm_toolchain_ar = "${toolchain_path}/bin/llvm-ar"
         String npm_toolchain_cc = "${toolchain_path}/bin/${temp_compiler_prefix}-clang"
