diff --git a/node_modules/@rnmapbox/maps/setup-jest.js b/node_modules/@rnmapbox/maps/setup-jest.js
index c4da658..b02440f 100644
--- a/node_modules/@rnmapbox/maps/setup-jest.js
+++ b/node_modules/@rnmapbox/maps/setup-jest.js
@@ -206,9 +206,3 @@ NativeModules.RNMBXChangeLineOffsetsShapeAnimatorModule = {
 NativeModules.RNMBXLogging = nativeModule({
   setLogLevel: jest.fn(),
 });
-
-// Mock for global AbortController
-global.AbortController = class {
-  signal = 'test-signal';
-  abort = jest.fn();
-};
