diff --git a/node_modules/nodejs-mobile-react-native/android/build.gradle b/node_modules/nodejs-mobile-react-native/android/build.gradle
index a6fac6f..423cb52 100644
--- a/node_modules/nodejs-mobile-react-native/android/build.gradle
+++ b/node_modules/nodejs-mobile-react-native/android/build.gradle
@@ -377,6 +377,7 @@ if ("1".equals(shouldRebuildNativeModules)) {
             delete fileTree(dir: "${rootProject.buildDir}/nodejs-native-assets-temp-build/nodejs-native-assets-${abi_name}/nodejs-project/node_modules/").matching {
                 include "**/*.node" // Look for all .node files
                 exclude "**/prebuilds/android-${temp_arch}/*" // Don't touch the correct prebuilds
+                exclude "**/build/Release/*" // Don't touch the prebuilds moved here from previous run of DetectCorrectPrebuilds step
             }
         }
 
