diff --git a/node_modules/nodejs-mobile-react-native/android/build.gradle b/node_modules/nodejs-mobile-react-native/android/build.gradle
index b1957a5..8fe2082 100644
--- a/node_modules/nodejs-mobile-react-native/android/build.gradle
+++ b/node_modules/nodejs-mobile-react-native/android/build.gradle
@@ -417,9 +417,9 @@ if ("1".equals(shouldRebuildNativeModules)) {
             if (nodeVersionResult.exitValue != 0) {
                 throw new GradleException("Failed to get node version: ${nodeVersionResult.errorOutput}")
             }
-            if (nodeVersionStdout.toString().trim() != "${_nodeVersionRequired}") {
-                throw new GradleException("nodejs-mobile-react-native requires Node.js version ${_nodeVersionRequired}, but found ${nodeVersionStdout.toString().trim()}")
-            }
+            // if (nodeVersionStdout.toString().trim() != "${_nodeVersionRequired}") {
+            //     throw new GradleException("nodejs-mobile-react-native requires Node.js version ${_nodeVersionRequired}, but found ${nodeVersionStdout.toString().trim()}")
+            // }
             inputs.file "${rootProject.buildDir}/nodejs-native-assets-temp-build/nodejs-native-assets-${abi_name}/copy.timestamp"
             outputs.dir "${rootProject.buildDir}/nodejs-native-assets-temp-build/nodejs-native-assets-${abi_name}/nodejs-project/"
             workingDir "${rootProject.buildDir}/nodejs-native-assets-temp-build/nodejs-native-assets-${abi_name}/nodejs-project/"
diff --git a/node_modules/nodejs-mobile-react-native/scripts/ios-build-native-modules.sh b/node_modules/nodejs-mobile-react-native/scripts/ios-build-native-modules.sh
index e743704..2938302 100644
--- a/node_modules/nodejs-mobile-react-native/scripts/ios-build-native-modules.sh
+++ b/node_modules/nodejs-mobile-react-native/scripts/ios-build-native-modules.sh
@@ -14,7 +14,7 @@ if [ "$CURRENT_NODE_VERSION" -ne "$DESIRED_NODE_VERSION" ]; then
   echo "nodejs-mobile-react-native requires Node.js version \
 $DESIRED_NODE_VERSION accessible from Xcode, but found \
 $(node -p 'process.versions.node')"
-  exit 1
+#  exit 1
 fi
 
 # This is our nodejs-project folder that was copied to the Xcode build folder
