diff --git a/node_modules/react-native-confirmation-code-field/esm/MaskSymbol.js b/node_modules/react-native-confirmation-code-field/esm/MaskSymbol.js
index 206af99..0c0fb8a 100644
--- a/node_modules/react-native-confirmation-code-field/esm/MaskSymbol.js
+++ b/node_modules/react-native-confirmation-code-field/esm/MaskSymbol.js
@@ -5,7 +5,7 @@ export const MaskSymbol = ({ isLastFilledCell, children: symbol, maskSymbol, del
     'use memo';
     const [visibleFlag, setFlag] = useState(true);
     const toggleVisibility = useCallback(() => {
-        setFlag((prev) => !prev);
+        setFlag(false);
     }, []);
     useTimeout(toggleVisibility, delay);
     useEffect(() => {
