diff --git a/node_modules/rpc-reflector/lib/is-message-port-like.js b/node_modules/rpc-reflector/lib/is-message-port-like.js
index 819d618..e2ca7d5 100644
--- a/node_modules/rpc-reflector/lib/is-message-port-like.js
+++ b/node_modules/rpc-reflector/lib/is-message-port-like.js
@@ -9,7 +9,7 @@ module.exports = isMessagePortLike
  */
 function isMessagePortLike(value) {
   if (typeof value !== 'object' || value === null) return false
-  if (typeof window !== 'undefined' && value instanceof window.MessagePort) {
+  if (typeof window !== 'undefined' && 'MessagePort' in window && value instanceof window.MessagePort) {
     return true
   } else if (
     typeof value.on === 'function' &&
diff --git a/node_modules/rpc-reflector/lib/validate-message.js b/node_modules/rpc-reflector/lib/validate-message.js
index 11c2b45..0cd2308 100644
--- a/node_modules/rpc-reflector/lib/validate-message.js
+++ b/node_modules/rpc-reflector/lib/validate-message.js
@@ -16,7 +16,7 @@ const objectStreamSuggest =
  */
 module.exports = function isValidMessage(msg) {
   let invalid = ''
-  if (Buffer.isBuffer(msg)) {
+  if (false) {
     invalid = `Received a Buffer (expected an array). ${objectStreamSuggest} `
   } else if (typeof msg === 'string') {
     invalid = `Received a string (expected an array). ${objectStreamSuggest} `
