diff --git a/node_modules/@comapeo/core/src/config-import.js b/node_modules/@comapeo/core/src/config-import.js
index fe8be63..5e83465 100644
--- a/node_modules/@comapeo/core/src/config-import.js
+++ b/node_modules/@comapeo/core/src/config-import.js
@@ -9,7 +9,7 @@ import { SUPPORTED_CONFIG_VERSION } from './constants.js'
 // Throw error if a zipfile contains more than 10,000 entries
 const MAX_ENTRIES = 10_000
 const MAX_ICON_SIZE = 10_000_000
-const ICON_NAME_REGEX = /([a-zA-Z0-9-]+)-([a-zA-Z]+)@(\d+)x\.[a-zA-Z]+$/
+const ICON_NAME_REGEX = /([a-zA-Z0-9-]+)-([a-zA-Z]+)(:?@(\d+)x)?\.[a-zA-Z]+$/
 
 /**
  * @typedef {yauzl.Entry} Entry
@@ -518,7 +518,7 @@ function parseIcon(filename, buf) {
   }
   const [_, name, size, pixelDensityStr] = matches
   const pixelDensity = Number(pixelDensityStr)
-  if (!(pixelDensity === 1 || pixelDensity === 2 || pixelDensity === 3)) {
+  if (pixelDensityStr && !(pixelDensity === 1 || pixelDensity === 2 || pixelDensity === 3)) {
     throw new Error(`Error loading icon. invalid pixel density ${pixelDensity}`)
   }
   if (!(size === 'small' || size === 'medium' || size === 'large')) {
diff --git a/node_modules/@comapeo/core/src/mapeo-project.js b/node_modules/@comapeo/core/src/mapeo-project.js
index 0bb67b3..2780ca9 100644
--- a/node_modules/@comapeo/core/src/mapeo-project.js
+++ b/node_modules/@comapeo/core/src/mapeo-project.js
@@ -1506,6 +1506,7 @@ export class MapeoProject extends TypedEmitter {
       return config.warnings
     } catch (e) {
       this.#l.log('error loading config', e)
+      console.error(e)
       this.#loadingConfig = false
       return /** @type Error[] */ []
     }
