diff --git a/node_modules/@comapeo/core/src/fastify-controller.js b/node_modules/@comapeo/core/src/fastify-controller.js
index a84c600..d7c748c 100644
--- a/node_modules/@comapeo/core/src/fastify-controller.js
+++ b/node_modules/@comapeo/core/src/fastify-controller.js
@@ -64,7 +64,14 @@ export class FastifyController {
 
   async #stopServer() {
     const { server } = this.#fastify
-    await promisify(server.close.bind(server))()
+
+    const closePromise = promisify(server.close.bind(server))()
+
+    // Need to force close some connections (see https://github.com/digidem/comapeo-mobile/issues/611#issuecomment-2368795563)
+    // We call this after `server.close()` as recommended by the Node docs (see https://nodejs.org/docs/latest-v20.x/api/http.html#servercloseidleconnections)
+    server.closeIdleConnections()
+
+    await closePromise
   }
 
   /**
