diff --git a/node_modules/@mapeo/core/src/mapeo-manager.js b/node_modules/@mapeo/core/src/mapeo-manager.js
index d138f56..29c23d5 100644
--- a/node_modules/@mapeo/core/src/mapeo-manager.js
+++ b/node_modules/@mapeo/core/src/mapeo-manager.js
@@ -94,7 +94,7 @@ export class MapeoManager extends TypedEmitter {
     )
     this.#db = drizzle(sqlite)
     migrate(this.#db, {
-      migrationsFolder: new URL('../drizzle/client', import.meta.url).pathname,
+      migrationsFolder: new URL('./node_modules/@mapeo/core/drizzle/client', import.meta.url).pathname,
     })
 
     this.#localPeers = new LocalPeers({ logger: this.#l })
diff --git a/node_modules/@mapeo/core/src/mapeo-project.js b/node_modules/@mapeo/core/src/mapeo-project.js
index 25bd118..cfbb9f4 100644
--- a/node_modules/@mapeo/core/src/mapeo-project.js
+++ b/node_modules/@mapeo/core/src/mapeo-project.js
@@ -102,7 +102,7 @@ export class MapeoProject {
     const sqlite = new Database(dbPath)
     const db = drizzle(sqlite)
     migrate(db, {
-      migrationsFolder: new URL('../drizzle/project', import.meta.url).pathname,
+      migrationsFolder: new URL('./node_modules/@mapeo/core/drizzle/project', import.meta.url).pathname,
     })
 
     ///////// 2. Setup random-access-storage functions
