diff --git a/node_modules/@mapeo/core/src/utils.js b/node_modules/@mapeo/core/src/utils.js
index bbfb30c..5e700a4 100644
--- a/node_modules/@mapeo/core/src/utils.js
+++ b/node_modules/@mapeo/core/src/utils.js
@@ -1,5 +1,5 @@
 import b4a from 'b4a'
-import { projectKeyToPublicId as keyToPublicId } from '@mapeo/crypto'
+import { keyToPublicId } from '@mapeo/crypto'
 
 /**
  * @param {String|Buffer} id
