diff --git a/node_modules/crc-universal/index.js b/node_modules/crc-universal/index.js
index 619d62d..438d710 100644
--- a/node_modules/crc-universal/index.js
+++ b/node_modules/crc-universal/index.js
@@ -1,11 +1,13 @@
 const fallback = require('./fallback')
 
-try {
-  const native = require('crc-native')
+module.exports = fallback
 
-  exports.crc32 = function crc32 (buffer) {
-    return buffer.byteLength <= 24 ? fallback.crc32(buffer) : native.crc32(buffer)
-  }
-} catch {
-  module.exports = fallback
-}
+// try {
+//   const native = require('crc-native')
+
+//   exports.crc32 = function crc32 (buffer) {
+//     return buffer.byteLength <= 24 ? fallback.crc32(buffer) : native.crc32(buffer)
+//   }
+// } catch {
+//   module.exports = fallback
+// }
