diff --git a/node_modules/fs-native-extensions/binding.gyp b/node_modules/fs-native-extensions/binding.gyp
index f1f518c..f91caeb 100644
--- a/node_modules/fs-native-extensions/binding.gyp
+++ b/node_modules/fs-native-extensions/binding.gyp
@@ -36,6 +36,13 @@
           './src/posix.c',
         ],
       }],
+      ['OS == "android" or OS=="ios"', {
+        'sources': [
+          './src/linux.c'
+        ],
+        'cflags': [ '-fPIC' ],
+        'ldflags': [ '-fPIC' ]
+      }]
     ],
   }]
 }
