diff --git a/node_modules/node-gyp-build/node-gyp-build.js b/node_modules/node-gyp-build/node-gyp-build.js
index 61b398e..ace6a1c 100644
--- a/node_modules/node-gyp-build/node-gyp-build.js
+++ b/node_modules/node-gyp-build/node-gyp-build.js
@@ -18,13 +18,17 @@ var uv = (process.versions.uv || '').split('.')[0]
 
 module.exports = load
 
-function load (dir) {
-  return runtimeRequire(load.resolve(dir))
+function load (dir, moduleName) {
+  return runtimeRequire(load.resolve(dir, moduleName))
 }
 
-load.resolve = load.path = function (dir) {
+load.resolve = load.path = function (dir, moduleName) {
   dir = path.resolve(dir || '.')
 
+  if (platform === 'android' && moduleName) {
+    dir = path.resolve(dir, 'node_modules', moduleName)
+  }
+
   try {
     var name = runtimeRequire(path.join(dir, 'package.json')).name.toUpperCase().replace(/-/g, '_')
     if (process.env[name + '_PREBUILD']) dir = process.env[name + '_PREBUILD']
