diff --git a/node_modules/typebox/build/format/idn-email.mjs b/node_modules/typebox/build/format/idn-email.mjs
index a9ce0c0..88f0712 100644
--- a/node_modules/typebox/build/format/idn-email.mjs
+++ b/node_modules/typebox/build/format/idn-email.mjs
@@ -1,4 +1,4 @@
-const IdnEmail = /^(?!.*\.\.)[\p{L}\p{N}!#$%&'*+/=?^_`{|}~-]+(?:\.[\p{L}\p{N}!#$%&'*+/=?^_`{|}~-]+)*@[\p{L}\p{N}](?:[\p{L}\p{N}-]{0,61}[\p{L}\p{N}])?(?:\.[\p{L}\p{N}](?:[\p{L}\p{N}-]{0,61}[\p{L}\p{N}])?)*$/iu;
+const IdnEmail = /^(?!.*\.\.)[a-zA-Z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-zA-Z0-9!#$%&'*+/=?^_`{|}~-]+)*@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/i;
 /** Returns true if the value is a Idn Email */
 export function IsIdnEmail(value) {
     return IdnEmail.test(value);
diff --git a/node_modules/typebox/build/guard/emit.mjs b/node_modules/typebox/build/guard/emit.mjs
index b9b7ece..ab936d3 100644
--- a/node_modules/typebox/build/guard/emit.mjs
+++ b/node_modules/typebox/build/guard/emit.mjs
@@ -2,7 +2,7 @@ import * as G from './guard.mjs';
 // ------------------------------------------------------------------
 // Identifier
 // ------------------------------------------------------------------
-const identifierRegExp = /^[\p{ID_Start}_$][\p{ID_Continue}_$\u200C\u200D]*$/u;
+const identifierRegExp = /^[a-zA-Z_$][a-zA-Z0-9_$\u200C\u200D]*$/;
 /** Returns true if this value is a valid JavaScript identifier */
 function IsIdentifier(value) {
     return identifierRegExp.test(value);
